core/src/server/http/http_response.cpp

```diff
@@ -272,9 +292,24 @@ void HttpResponse::SendResponse(engine::io::RwBase& socket) {
                        (request_.IsFinal() ? kClose : kKeepAlive));
   }
   for (const auto& cookie : cookies_) {
-    header.append(USERVER_NAMESPACE::http::headers::kSetCookie);
-    header.append(kKeyValueHeaderSeparator);
+    const std::size_t old_size = header.size();
+
+    header.resize_and_overwrite(
+        old_size +
+            USERVER_NAMESPACE::http::headers::kSetCookie
```

Comment:
```suggestion
            static_cast<std::string_view>(USERVER_NAMESPACE::http::headers::kSetCookie). size()
```

Так попривычнее и чуть красивее